news/2024/2024-09-18-osumania-4k-world-cup-2024-semifinals-recap.md

```diff
@@ -0,0 +1,128 @@
+---
+layout: post
+title: "osu!mania 4K World Cup 2024: Semifinals Recap"
+date: 2024-09-18 21:30:00 +0000
+---
+
+The walls are slowly closing in for the remainder teams, and only the top 6 are making it to the Finals. It's time to lock in, review the previous week, and prepare thy selves for the upcoming showdown of epic proportions.
+
+![](/wiki/Tournaments/MWC/2024_4K/img/mwc4k2024-banner.jpg)
+
+At this point in the tournament, teams cannot get away with just raw skill, they must watch and analyse their opponents, and make sure they make all the right moves at any given point. Matches slowly turn into masterful showcases of not just skill, but the team's knowledge of the opposing team and their strengths and weaknesses. Each match is played like a delicate dance of tango, where nothing but perfect strategy and execution leads to a victory, with teams constantly looking for any smallest missteps by their opponents.
+
+Let us ponder no more and take a look at the best matches of this week, and see whether teams were doing the tango, or stumbled throughout their matches.
+
+## Mappool showcase
+
+<style>
+    .osu-md--news .audio-player {
+        max-width: inherit;
+    }
+</style>
+
+[**Download the pool here! (212 MB)**](https://osu.ppy.sh/beatmaps/packs/P260)
+
+<div align="center" class="osu-md__paragraph">
+    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://player.twitch.tv/?video=2246114290&parent=osu.ppy.sh&autoplay=false" frameborder="0" allowfullscreen="true" scrolling="no"></iframe>
+</div>
+
+With Semifinals comes the second and last time we see two new slots put into the mix, first being Stamina (RC3), with [Black Chocolates](https://osu.ppy.sh/beatmapsets/1433708#mania/3401103) By [Shoegazer](https://osu.ppy.sh/users/2520707). The stamina pick is usually similar to the Jumpstream pick, with the big difference being that it plays differently and goes on for much longer, which is a large reason why it's named Stamina in the first place.
```

Comment:
these seem to be skill sets or map types, and not the mappool slots we would typically capitalise

```suggestion
With Semifinals comes the second and last time we see two new slots put into the mix, first being stamina (RC3), with [Black Chocolates](https://osu.ppy.sh/beatmapsets/1433708#mania/3401103) by [Shoegazer](https://osu.ppy.sh/users/2520707). The stamina pick is usually similar to the jumpstream pick, with the big difference being that it plays differently and goes on for much longer, which is a large reason why it's named stamina in the first place.
```